src/TLabel.cpp

```diff
@@ -131,12 +164,36 @@ void TLabel::mouseMoveEvent(QMouseEvent* event)
 {
     if (forwardEventToMapper(event)) {
         return;
+    } else if (mpHost) {
+        TEvent tmpMoveParams = mMoveParams;
+        for (auto button : mMouseButtons.keys()) {
+            if (button & event->buttons()) {
+                tmpMoveParams.mArgumentList.append(mMouseButtons.value(button));
+                tmpMoveParams.mArgumentTypeList.append(ARGUMENT_TYPE_STRING);
+            }
+        }
+        mpHost->getLuaInterpreter()->callEventHandler(mMove, tmpMoveParams);
+        event->accept();
+        return;
     }
+
+    QWidget::mouseMoveEvent(event);
 }
 
 void TLabel::wheelEvent(QWheelEvent* event)
 {
-    static_cast<void>(forwardEventToMapper(event));
+    if (forwardEventToMapper(event))
+        return;
+    else if (mpHost) {
+        TEvent tmpWheelParams = mWheelParams;
+        tmpWheelParams.mArgumentList.append(QString::number(event->angleDelta().x() / 8));
+        tmpWheelParams.mArgumentTypeList.append(ARGUMENT_TYPE_NUMBER);
+        tmpWheelParams.mArgumentList.append(QString::number(event->angleDelta().y() / 8));
+        tmpWheelParams.mArgumentTypeList.append(ARGUMENT_TYPE_NUMBER);
+        mpHost->getLuaInterpreter()->callEventHandler(mWheel, tmpWheelParams);
+        event->accept();
```

Comment:
I've got my mouse hack into my currently running Linux kernel - I'll look forward to verifying **both** mouse wheels work. 😈